src/components/Book.js

```diff
@@ -1,17 +1,30 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import { bookRemoved } from '../Redux/books/bookSlice';
 
-const Book = ({ title, author }) => (
-  <div className="book">
-    <h1>{title}</h1>
-    <p>{author}</p>
-    <button type="button">Remove</button>
-  </div>
-);
+const Book = ({ title, author, id }) => {
+  const dispatch = useDispatch();
+
+  const handleRemoveBook = (id) => {
+    dispatch(bookRemoved(id));
+  };
+
+  return (
+    <div className="book">
+      <h1>{title}</h1>
+      <p>{author}</p>
+      <button type="button" onClick={() => handleRemoveBook(id)}>
+        Remove
+      </button>
```

Comment:
- [ ] Removing a book is not working as expected, because, you're not rendering the store data so you'll tell if a book is removed from the state. Please work on that 👍🏽